### PR TITLE
Preliminary specfile stuff.

### PR DESCRIPTION
**Pull request by ralphbean**

### CODE REVIEW

**Comment by lmacken** [comment]
:yum:

File: bodhi.spec [hunk 1]
```diff
@@ -0,0 +1,646 @@
...
+
+# External resources
+BuildRequires:   python-bugzilla
+BuildRequires:   python-simplemediawiki
+BuildRequires:   fedmsg
+
+BuildRequires:   python-sphinx
+
+# For the bodhi-client
+BuildRequires:   python-click
```
Comment (pypingou) [inline bodhi.spec]: Should this be moved to the -client subpackage then?

File: apache/bodhi.wsgi [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import sys
+sys.stdout = sys.stderr
+
+import os
+os.environ['PYTHON_EGG_CACHE'] = '/var/www/.python-eggs'
```
Comment (pypingou) [inline apache/bodhi.wsgi]: Is this a file/location that bodhi should read/write to?

**Comment by pypingou** [comment]
Couple of questions otherwise :+1: for me

**Pull request merged**